plugins/amazonq/mynah-ui/src/mynah-ui/ui/connector.ts

```diff
@@ -513,7 +513,7 @@ export class Connector {
                 this.featureDevChatConnector.onOpenDiff(tabID, filePath, deleted)
                 break
             case 'codetest':
-                this.codeTestChatConnector.onFileClick(tabID, filePath, deleted, messageId)
+                this.codeTestChatConnector.onFormButtonClick(tabID, messageId ?? '', {id: "utg_view_diff"})
```

Comment:
This is fine for now but can you please add a TODO to clean this up? Ideally `codeTestChatConnector` should have a method called `onOpenDiff` instead of calling `onFormButtonClick` which should only be used by form buttons.